tests/sentinel/tests/11-sentinel-port-reset.tcl

```diff
@@ -0,0 +1,51 @@
+source "../tests/includes/init-tests.tcl"
+
+proc check_sentinel_port_collision {} {
+	#make sure no two sentinel has same port
+	foreach_sentinel_id id {
+		set sentinel_ports {}
+		foreach sentinel [S $id SENTINEL SENTINELS mymaster] {
+			set sentinel_port [dict get $sentinel port]
+		    if {$sentinel_port in $sentinel_ports} {
+		        fail "Sentinel port collision detected, port number is $sentinel_port."
+		    } 
+		    lappend sentinel_ports $sentinel_port
+		}
+	}
+}
+
+test "Start/Stop sentinel on same port should not cause port collision in other instances view" {
+	set sentinel_id [expr $::instances_count-1]
+	set org_runid [S $sentinel_id SENTINEL MYID]
+
+	kill_instance sentinel $sentinel_id
+
+	after 5000
```

Comment:
@hwware Do we really need such a long delay here (and in the other locations)?

Reply:
@yossigo we don't have to, the purpose of wait here is to make sure other sentinel get updated information from the pub/sub hello message, the interval is 2s. i think 3s should be enough, or what we can do is continusly waiting for other sentinel and as soon as the pub/sub message get updated we are good to go.

Reply:
@hwware I think it's worth taking the later approach, Sentinel tests are already pretty slow.

Reply:
@hwware As mentioned in #8710, maybe we need to make some `#define`'s configurable at least for testing purposes because the Sentinel tests are slow as they are. Does that make sense?

Reply:
@yossigo I totally agree with you, and I will create a separated PR for this configuration. Thanks